### PR TITLE
Add SUM

### PR DESCRIPTION
**Pull request by 5053s**
Initial request for review to add Sumcoin to swap.core - https://www.sumcoin.org

### CODE REVIEW

**Comment by 5053s** [comment]
> it works?

Hi you have a concern?

File: src/swap.auth/sum.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import SwapApp from 'swap.app'
+
+
+const login = (_privateKey, app) => {
+  SwapApp.required(app)
+
+  const storageKey = `${app.network}:ltc:privateKey`
```
Comment (noxonsu) [inline src/swap.auth/sum.js:7]: ltc?

Comment (5053s) [reply src/swap.auth/sum.js:7]: Whoops.   I used ltc as a template.  My oversight.  Let me change this.
Was this the cause of the build error?  I only see a Node version error

Comment (5053s) [reply src/swap.auth/sum.js]: There appears to be some issue with the core itself.  When Travis builds it's returning eos and btc errors.   Please advise.

https://travis-ci.com/github/swaponline/swap.core/jobs/319035769#L461

**Comment by sumcoinlabs** [comment]
Hi, can this be reviewed?

**Comment by 5053s** [comment]
@shendel @noxonsu  Can I get a review?

**Comment by noxonsu** [comment]
Sorry for the delayed response. All looks good. I've created a Frontend  build ( https://github.com/swaponline/MultiCurrencyWallet ) with this fork and it doesn't have backward compatibility issues (i've check BTC-USDT atomic swap, not SUM-*). You can start futher development - add SUM to the UX. Unfortunately i can't merge this PR to the master branch because i am not sure that our comercial clients want to see SUM on their sites. Anyway i am very happy that someone understand how to use our core library and hope that you will continue development in your own repository (next step: clone https://github.com/swaponline/MultiCurrencyWallet and integrate SUM there ). Please contact me when your UX part will be ready, i'll check and make promo in our twitter. Best regards!

**Comment by 5053s** [comment]
@noxonsu Thank you for the reply.   I have a fork of UX.  If you can give me a short list of paths where SUM needs to be added, it would me a lot for efficiency reasons.   Please let me know where in the code needs attention given

**Pull request closed without merging**